### DISCUSSION TITLE
Not recognising latest NodeJS builds. Warnings on updating packages.

### DISCUSSION THREAD

**Discussion by OrigSorceror** [dgreif/ring#957] [General]
Not that it affects anything greatly, everything still works.  but when installing updates i get the following :

npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: 'homebridge-ring@11.0.4',
npm WARN EBADENGINE   required: { node: '^16', homebridge: '>=1.4.0' },
npm WARN EBADENGINE   current: { node: 'v18.2.0', npm: '8.9.0' }
npm WARN EBADENGINE }
npm WARN deprecated request-promise@4.2.6: request-promise has been deprecated because it extends the now deprecated request package, see https://github.com/request/request/issues/3142
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142

just an FYI at this stage ;)

Regards, 

Alan

**Comment by dgreif** [comment]
It looks like you have node 18 installed.  Node 16 is the current LTS version, and is the only version that will work properly with the Ring plugin

**Reply by OrigSorceror** [reply]
Yup Reverted to Node 16
